packages/thirdweb/src/react/web/wallets/smartWallet/locale/ru.ts

```diff
@@ -0,0 +1,13 @@
+import type { SmartWalletLocale } from "./types.js";
+
+const smartWalletLocalRu: SmartWalletLocale = {
```

Comment:
This is only one of the language files we have, would you be open to updating the other ones too?